biorbd_optim/gui/plot.py

```diff
@@ -160,15 +167,25 @@ def __create_plots(self):
                 self.plot_func[variable][i] = nlp["plot"][variable]
 
                 mapping = self.plot_func[variable][i].phase_mappings.map_idx
-                for k in mapping:
+                for ctr, k in enumerate(mapping):
                     ax = axes[k]
                     if k < len(self.plot_func[variable][i].legend):
                         axes[k].set_title(self.plot_func[variable][i].legend[k])
-                    ax.grid(color="k", linestyle="--", linewidth=0.5)
+                    ax.grid(color="k", linestyle="-", linewidth=0.15)
                     ax.set_xlim(0, self.t[-1][-1])
                     if nlp["plot"][variable].ylim:
                         ax.set_ylim(nlp["plot"][variable].ylim)
-
+                    elif self.use_bounds_as_ylimit and nlp["plot"][variable].bounds is not None:
+                        if nlp["plot"][variable].bounds.type != InterpolationType.CUSTOM:
```

Comment:
Please inverse this if (don't use "if not" unless necessary)
so : 
`if ... == ...`